clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -186,7 +186,11 @@ public DaVinciBackend(
           new VeniceWriterFactory(backendProps.toProperties(), pubSubClientsFactory.getProducerAdapterFactory(), null);
       String instanceName = Utils.getHostName() + "_" + Utils.getPid();
       int derivedSchemaID = backendProps.getInt(PUSH_STATUS_STORE_DERIVED_SCHEMA_ID, 1);
-      pushStatusStoreWriter = new PushStatusStoreWriter(writerFactory, instanceName, derivedSchemaID);
+      Map<String, VeniceWriterFactory> writerFactoryMap = new HashMap<>(1);
```

Comment:
Why (1)?  It seems we don't do this in other similar spots in this PR, but why here?

Reply:
Using (1) here is for having fixed `clusterName` for all stores in this `DavinciBackend`, as I changed the constructor of `PushStatusStoreWriter` for using `clusterName` to `VeniceWriterFactory` map and cluster discovery supplier function. For controller, it will use proper cluster discover supplier function. As we want to have a PubSubClientFactory for all stores regardless which cluster a specific store belongs to, we just using a placeholder clusterName here as only one entry for this map, and each store discovery logic will return that clusterName to help it find that specific VencieWriterFactory for every store.